content/extend/how-can-i-contribute.md

```diff
@@ -60,26 +60,19 @@ Usually, it takes less time to fix a bug if the developer knows how to find it.
 
 Taiga will always be glad to receive code patches to update, fix or improve its code.
 
-If you know how to improve our code base or you found a bug, a security vulnerability or a performance issue and you think you can solve it, we will be very happy to accept your pull-request. If your code requires considerable changes, we recommend you first  talk to us directly. We will find the best way to help. If you would like to implement it by yourself consider if it's a small or a big change.
+If you know how to improve our code base or you found a bug, a security vulnerability or a performance issue and you think you can solve it, we will be very happy to accept your contribution.
 
-Besides, you'd probably like to make yourself confortable with the [development environment](http://docs.taiga.io/setup-development.html). If you find some problems, please let us know in our [mailing list](https://groups.google.co.uk/d/forum/taigaio).
+You probably want to make yourself comfortable with the development environment first, but we have defined a concrete workflow for these kinds of contributions:
 
-### Small Changes
+1. **It’s essential to have an issue to associate your PR with***, so if it does not exist, you will have to create it in Taiga (preferably) or in GitHub.
 
-They can be crafted and submited to the [Github repositories](https://github.com/kaleidos-ventures) as a _Pull Request_.
+   This issue should have a complete description of the functionality you’d like to develop or the bug you intend to solve, how it should work and for which profiles.
+2. **In the issue, let us know your intention to work on it**, and we will contact you to determine the priority and needs. 
 
-### Major Changes
+   The functionality will require some wireframes or ideas to be developed correctly. A good UX its an essential part of Taiga success. If the issue requires it, the Taiga team will participate in its development with you, providing the necessary designs and wireframes.
```

Comment:
This assumes that we will only receive code patches or the persona will not be able to suggest a UI. I'd suggest using a general tone: 

```
The functionality might require some brainstorming to be developed correctly. A good code and UI are essential part of Taiga success. If the issue requires it, the Taiga team will participate in its development with you, providing the necessary designs, wireframes or code suggestions.
```